dashboard/app/helpers/allowed_hostname_helper.rb

```diff
@@ -4,107 +4,137 @@
 # - APIs that require API keys should be documented as such.  These risk student exposure to the API keys.
-#
+# This list was last updated 9/3/2025. There was a best effort to determine which APIs used keys and which were public.
+# If an API is not specified as either, it means we could not determine if it required a key or not.
+# API descriptions were generated by AI and spot-checked for accuracy, but may contain errors.
+
 module AllowedHostnameHelper
-  # 'code.org' is included so applab apps can access the tables and properties of other applab apps.
-  ALLOWED_HOSTNAME_SUFFIXES = %w(
-    api.amadeus.com
-    api.arasaac.org
-    api.blizzard.com
-    api.census.gov
-    api.coinlayer.com
-    api.datamuse.com
-    api.energidataservice.dk
-    api.exchangeratesapi.io
-    api.fda.gov
-    api.football-data.org
-    api.foursquare.com
-    api.github.com
-    api.mathjs.org
-    api.mojang.com
-    api.nal.usda.gov
-    api.nasa.gov
-    api.nookipedia.com
-    api.opencagedata.com
-    api.open-notify.org
-    api.open-meteo.com
-    api.openrouteservice.org
-    api.openweathermap.org
-    api.pegelalarm.at
-    api.quotable.io
-    api.randomuser.me
-    api.rebrandly.com
-    api.scripture.api.bible
-    api.scryfall.com
-    api.spoonacular.com
-    api.sportsdata.io
-    api.si.edu
-    api.spacexdata.com
-    api.spotify.com
-    api.themoviedb.org
-    api.thingspeak.com
-    api.waqi.info
-    api.weather.gov
-    api.weatherapi.com
-    api.wolframalpha.com
-    api.zippopotam.us
-    bnefoodtrucks.com.au
-    ch.tetr.io
-    code.org
-    covidtracking.com
-    cryptonator.com
-    currencyapi.com
-    data.austintexas.gov
-    data.cityofchicago.org
-    data.gv.at
-    data.nasa.gov
-    data.weather.gov.hk
-    dataservice.accuweather.com
-    deckofcardsapi.com
-    distanza.org
-    githubusercontent.com
-    googleapis.com
-    grobchess.com
-    hubblesite.org
-    images-api.nasa.gov
-    itunes.apple.com
-    io.adafruit.com
-    isenseproject.org
-    lakeside-cs.org
-    maker.ifttt.com
-    moneyconvert.net
-    myschoolapp.com
-    native-land.ca
-    newsapi.org
-    noaa.gov
-    numbersapi.com
-    open.mapquestapi.com
-    openlibrary.org
-    opentdb.com
-    perenual.com
-    pixabay.com
-    pokeapi.co
-    pro-api.coinmarketcap.com
-    qrng.anu.edu.au
-    quandl.com
-    random.org
-    rejseplanen.dk
-    restcountries.com
-    roblox.com
-    runescape.com
-    serpapi.com
-    sessionserver.mojang.com
-    stats.minecraftservers.org
-    swapi.dev
-    textures.minecraft.net
-    thecatapi.com
-    thedogapi.com
-    theunitedstates.io
-    transitchicago.com
-    vpic.nhtsa.dot.gov
-    wikipedia.org
-    worldclockapi.com
-    worldtimeapi.org
-    xeno-canto.org
-  ).freeze
+  ALLOWED_HOSTNAME_SUFFIXES = [
+    # === ENTERTAINMENT ===
+    # SECURITY: All are public APIs with no authentication required for basic usage
```

Comment:
Does line 14 apply to all the APIs list below or just the ones from 'ENTERTAINMENT'?

Reply:
Nit: the `SECURITY` doc seems to apply to only the section it is underneath. Maybe add a little formatting to make this more visible? (e.g., add a newline between sections or indent the security statement)